packages/fx-core/src/component/driver/teamsApp/copyAppPackageForSPFx.ts

```diff
@@ -55,12 +74,31 @@ export class CopyAppPackageForSPFxDriver implements StepDriver {
       : path.join(context.projectPath, copyAppPackageArgs.spfxFolder);
     const spfxTeamsPath = `${spfxFolder}/teams`;
     await fs.copyFile(appPackagePath, path.join(spfxTeamsPath, "TeamsSPFxApp.zip"));
+    context.addSummary(
+      getLocalizedString(
+        "driver.teamsApp.summary.copyAppPackageSuccess",
+        appPackagePath,
+        path.join(spfxTeamsPath, "TeamsSPFxApp.zip")
+      )
+    );
 
-    for (const file of await fs.readdir(`${spfxFolder}/teams`)) {
+    for (const file of await fs.readdir(spfxTeamsPath)) {
       if (file.endsWith("color.png") && pictures.color) {
         await fs.writeFile(path.join(spfxTeamsPath, file), pictures.color);
+        context.addSummary(
```

Comment:
If there're multiple web parts, the summary will be added multiple times with same content. Better only print once